ReactWindows/ReactNative/Bridge/ReactApplicationContext.cs

```diff
@@ -0,0 +1,13 @@
+﻿
+namespace ReactNative.Bridge
+{
+    /// <summary>
+    /// This class serves as a base class on top of <see cref="ReactContext"/>.
+    /// </summary>
+    public class ReactApplicationContext : ReactContext
```

Comment:
Why do need this class?  It has no behavior.  Maybe we should deviate from the Java implementation here a bit.